control-plane/gateways/service_test.go

```diff
@@ -83,9 +84,9 @@ func Test_meshGatewayBuilder_Service(t *testing.T) {
 					Ports: []corev1.ServicePort{
 						{
 							Name: "wan",
-							Port: int32(443),
```

Comment:
should we have a separate path in here showing that when the gcc is not nil and port is not set that we default to 443 as the port?

Reply:
Great idea! On it now